### PR TITLE
legacy config service shim

### PR DESCRIPTION
**Pull request by bbaldino**
A shim for the old `ConfigurationService`, since some libraries that JVB, for example, depend on still use it.  This shim implements the old interface, but reads from the legacy config file via the new config classes.

### CODE REVIEW

**Comment by bbaldino** [comment]
Found an issue here: the old configuration service fell back to checking for the path in system properties.  Working on a fix for that.

File: jicoco-kotlin/src/main/kotlin/org/jitsi/config/LegacyConfigurationServiceShim.kt [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+    override fun reloadConfiguration() {
+        legacyShimConfig.reload()
+    }
+
+    override fun removeProperty(propertyName: String?) {
+        throw Exception("Unsupported")
+    }
+
+    override fun setProperties(properties: MutableMap<String, Any>?) {
```
Comment (bgrozev) [inline jicoco-kotlin/src/main/kotlin/org/jitsi/config/LegacyConfigurationServiceShim.kt]: We might need this one (and some others) for jigasi. I guess we'll find out when we try to update jicoco there.

Comment (bgrozev) [reply jicoco-kotlin/src/main/kotlin/org/jitsi/config/LegacyConfigurationServiceShim.kt]: Actually, jigasi would be using the old config through jitsi/libjitsi, right?

Comment (bbaldino) [reply jicoco-kotlin/src/main/kotlin/org/jitsi/config/LegacyConfigurationServiceShim.kt]: yup, exactly.

**Pull request merged**